src-plugins/mscExportVideo/mscExportVideo.cpp

```diff
@@ -347,6 +372,30 @@ void ExportVideo::handleWidgetDisplayAccordingToType(int index)
         d->qualityLabel->hide();
         break;
     }
+    case FFMPEG:
+    {
+        // Video
+        d->exportDialog->selectFile("video.avi");
+        d->frameRateSpinBox->show();
+        d->frameRateLabel->show();
+        d->subsamplingComboBox->hide();
+        d->subsamplingLabel->hide();
+        d->qualityComboBox->hide();
+        d->qualityLabel->hide();
+        break;
+    }
+    case MPEG2:
+    {
+        // Video
+        d->exportDialog->selectFile("video.avi");
+        d->frameRateSpinBox->hide();
+        d->frameRateLabel->hide();
+        d->subsamplingComboBox->hide();
+        d->subsamplingLabel->hide();
+        d->qualityComboBox->hide();
+        d->qualityLabel->hide();
+        break;
+    }
     }
 }
 
```

Comment:
In acccordance to the DRY (Don't Repeat Yourself) principle, 
maybe you could merge the 2 cases FFMPEG and MPEG2.

Reply:
Even if it's not exactly the same code? there are some lines using `show()` and others `hide()`. As you want

Reply:
Ok my bad...